### PR TITLE
feat: [a/b]show welcome page when user didn't open valid project

### PR DESCRIPTION
**Pull request by HuihuiWu-Microsoft**
image

### CODE REVIEW

File: packages/vscode-extension/src/handlers.ts [hunk 1]
```diff
@@ -105,10 +105,27 @@ export async function activate(): Promise<Result<Void, FxError>> {
...
+    const expService = exp.getExpService();
+    if (expService) {
+      if (
+        !validProject &&
+        (await expService.getTreatmentVariableAsync(
+          TreatmentVariables.VSCodeConfig,
+          TreatmentVariables.SidebarWelcome,
+          true
+        ))
+      ) {
```
Comment (1openwindow) [inline packages/vscode-extension/src/handlers.ts:122]: the if condition looks strange. should we check !validProject first, then getTreatmentVariableAsync().

Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/src/handlers.ts:122]: I think the getTreatmentVariableAsync will not be executed if !validProject returns false

**Comment by jaeyonglee8687** [comment]
A couple of changes I may require.

1. Remove the 'No project created' next to the Teams toolkit title. The space won't be used to represent the status of the toolkit.

2. Update the string: "To start Teams app development experience, create a new Teams app or explore our samples. For more information visit our documentation."

3. Give some vertical margin space between the first two primary actions and the last one (open folder) if applicable. 

See the UI design here: [Figma](https://www.figma.com/proto/qqVegbnWWBsqHhR8Qijs4O/Teams-Toolkit-GA?page-id=87%3A33391&node-id=427%3A59166&viewport=394%2C48%2C0.97&scaling=min-zoom&starting-point-node-id=427%3A59166)

**Review by 1openwindow — APPROVED** [review]

**Comment by HuihuiWu-Microsoft** [comment]
> A couple of changes I may require.
> 
> 1. Remove the 'No project created' next to the Teams toolkit title. The space won't be used to represent the status of the toolkit.
> 2. Update the string: "To start Teams app development experience, create a new Teams app or explore our samples. For more information visit our documentation."
> 3. Give some vertical margin space between the first two primary actions and the last one (open folder) if applicable.
> 
> See the UI design here: [Figma](https://www.figma.com/proto/qqVegbnWWBsqHhR8Qijs4O/Teams-Toolkit-GA?page-id=87%3A33391&node-id=427%3A59166&viewport=394%2C48%2C0.97&scaling=min-zoom&starting-point-node-id=427%3A59166)

Updated #1 and #2, but didn't find a way to implement #3.

**Pull request merged**